astropy/modeling/blackbody.py

```diff
@@ -48,6 +48,26 @@
 >>> flux_nu  # doctest: +FLOAT_CMP
 <Quantity [  4.25135927e-123,  2.36894060e-005] erg / (cm2 Hz s sr)>
 
+Alternatively, the same results for ``flux_nu`` can be computed using ``BlackBody1D``
+from :class:`BlackBody1D` with blackbody representation as a model. The difference between
+this and the former approach is in one addition step as outlined as follows:
+
+>>> import numpy as np
+>>> from astropy import constants as const
+>>> from astropy import units as u
+>>> from astropy.modeling import models
+>>> temperature = 5000 * u.K
+>>> extra = const.sigma_sb * temperature ** 4 / np.pi
+>>> extra.cgs  # doctest: +FLOAT_CMP
+<Quantity 1.12808367e+10 g / s3>
```

Comment:
When we're talking about flux, `g / s3` does not really mean anything useful. Maybe `extra.to(equivalent_flux_unit)` that gives the same number would be better. Either that, or just remove this print out from docstring.

Reply:
I think I have to manually change the cgs units so they conform with the standard ones... Otherwise, some CI checks are not passed using the `.to(equivalent_flux_unit)` syntax.